src/euphorie/client/docx/views.py

```diff
@@ -158,6 +157,7 @@ def get_risks_for(self, sql_module):
                 u'measures': measures,
                 u'epilogue': u'',
                 u'justifiable': sql_risk.identification,
+                u'number': sql_risk.number,
```

Comment:
Should we just add to the dict the `sql_risk` and then evaluate the number when needed?